Git repo caching (#6901)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/account-settings.md

## Git repository caching <Lifecycle status="enterprise" />

:::important repo caching enabled by default

> dbt Cloud Git repository caching is enabled by default for all new Enterprise accounts. To prevent dbt Cloud from caching your repo content, disable the feature prior to integrating with Git.

Reviewer: I wonder if we should mention why they need to disable prior to integrating with git? Will they need to disconnect and reconnect to Git or something?

> dbt Cloud Git repository caching is enabled by default for all new Enterprise accounts. To prevent dbt Cloud from caching your repo content, disable the feature prior to integrating with Git.

Author: ```suggestion
dbt Cloud Git repository caching is enabled by default for all new Enterprise accounts. Repos are cached only after a successful production run. Disabling repo caching after a completed run will result in the cache expiring after 8 days. 
```

> dbt Cloud Git repository caching is enabled by default for all new Enterprise accounts. Repos are cached only after a successful production run. Disabling repo caching after a completed run will result in the cache expiring after 8 days. 

Reviewer: ```suggestion
dbt Cloud Git repository caching is enabled by default for all new Enterprise accounts. Repos are cached only after a successful production run. Id you disable repo caching after a completed run, you'll still see the cache for 8 days. 
```

> dbt Cloud Git repository caching is enabled by default for all new Enterprise accounts. Repos are cached only after a successful production run. Disabling repo caching after a completed run will result in the cache expiring after 8 days. 

Author: ```suggestion
dbt Cloud Git repository caching is enabled by default for all new Enterprise accounts. Repos are cached only after a successful production run. If you disable repo caching after a successful production run, the cache will be deleted after 8 days. 
```
